distributed/worker.py

```diff
@@ -1971,6 +1975,29 @@ def run_coroutine(self, comm, function, args=(), kwargs=None, wait=True):
         return run(self, comm, function=function, args=args, kwargs=kwargs,
                    wait=wait)
 
+    @gen.coroutine
+    def plugin_add(self, comm=None, plugin=None, name=None):
```

Comment:
There are currently two ways of specifying the plugin name. I wonder if there really is a use case for the name attribute of a Plugin object.

Reply:
No strong opinion here.

Reply:
I definitely have use cases for the name attribute.  Often the author of a plugin knows more about how it should be used than the user of it.